src/test/java/com/marklogic/mgmt/admin/InitializeMarkLogicTest.java

```diff
@@ -27,7 +29,23 @@ public class InitializeMarkLogicTest extends AbstractMgmtTest {
      * to ensure no errors are thrown from bad JSON.
      */
     @Test
-    public void initAgainstAnAlreadyInitializedMarkLogic() {
-        adminManager.init();
+    void initAgainstAnAlreadyInitializedMarkLogic() {
+		assertDoesNotThrow(() -> adminManager.init());
     }
+
+	@Test
+	void withNullUsername() {
+		String originalUsername = adminConfig.getUsername();
```

Comment:
Use a try/finally block for each test to ensure that if the test fails, the username/password is still reverted.